src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6046,4 +6046,19 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_DuplicateRecordConstructor" xml:space="preserve">
     <value>There cannot be a primary constructor and a member constructor with the same parameter types.</value>
   </data>
+  <data name="ERR_InvalidWithReceiverType" xml:space="preserve">
+    <value>The receiver of a `with` expression must have a valid non-void type.</value>
+  </data>
+  <data name="ERR_NoSingleWithMethod" xml:space="preserve">
+    <value>The 'with' expression requires the receiver type '{0}' to have a single accessible non-inherited instance method named "With".</value>
+  </data>
+  <data name="ERR_ContainingTypeMustDeriveFromWithReturnType" xml:space="preserve">
+    <value>The type of the 'with' expression receiver, '{0}', does not derive from the return type of the 'With' method, '{1}'.</value>
```

Comment:
Man this is wordy and unreadable. Consider something simpler like `The return type of the 'With' method, '{0}', does not derive from '{1}'.`

Reply:
Which, is not much simpler, so if you have suggestions on making it better I'd be happy to listen.